packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -769,6 +769,7 @@ const VirtualListBaseFactory = (type) => {
 			delete rest.rtl;
 			delete rest.spacing;
 			delete rest.updateStatesAndBounds;
+			delete rest.isVerticalScrollbarVisible;
```

Comment:
Yeah I tried to narrow it to only use `Scroller`. You can check in `Scrollable` for the name of `ScrollerBase`, but they may be too restrictive. I think it would be better for the Wrapped component to opt in so they have the option to fix the problem themselves. 

I also tried to use React's new context API, but that causes our unit tests to fail because Enzyme hasn't updated yet.

Reply:
I got it.